synchronization/src/main/java/de/cyface/synchronization/HttpConnection.java

```diff
@@ -70,10 +70,14 @@
      * The boundary to be used in the Multipart request to separate data.
      */
     final static String BOUNDARY = "---------------------------boundary";
+    /**
+     * The string which is used for a line feed.
+     */
+    final static String LINE_FEED = "\r\n";
     /**
      * The tail to be used in the Multipart request to indicate that the request end.
      */
-    final static String TAIL = "\r\n--" + BOUNDARY + "--\r\n";
```

Comment:
The LINE_FEED which I removed here is actually part of the filePart, thus, I moved it to the file part generation method.